lib/datadog/appsec/contrib/devise/patcher.rb

```diff
@@ -36,7 +36,10 @@ def patch_rememberable_strategy
             return unless ::Devise::STRATEGIES.include?(:rememberable)
 
             # Rememberable strategy is required in autoloaded Rememberable model
-            ::Devise::Models::Rememberable # rubocop:disable Lint/Void
+            # Ensure Rememberable is loaded here, we call module_eval to
+            # avoid a Ruby warning when run with -w
+            ::Devise::Models::Rememberable.module_eval do
+            end
```

Comment:
I would not add this to our library code - this is purely for testing, and I think we can keep it in tests?